Documentation/known-issues.md

```diff
@@ -0,0 +1,61 @@
+Known issues & workarounds
+==========================
+
+This document outlines the known issues and workarounds for the current state of 
+the CLI tools. Issues will also have a workaround and affects sections if necessary. You can use this page to 
+get information and get unblocked. 
+
+This page lists only the major issues that block users in doing their everyday tasks and that affect all or 
+most of the commands in the CLI tools. For minor issues, your best bet is still to check out the issues in 
+this repo. 
+
+# Current issues (12/23/2015)
+
+## El Capitan support
+El Capitan's RID is not supported properly currently. 
+
+**Issues tracking this:** 
+
+* [#498](https://github.com/dotnet/cli/issues/498)
+* [#291](https://github.com/dotnet/cli/issues/291)
+
+**Affects:** most of the commands, but more than likely you will not be able to 
+use `dotnet compile` and `dotnet-run` on El Capitan. For others, there is a 
+workaround.
+
+**Workaround:** use the --runtime switch with the value of `osx.10.11-x64` in 
+`dotnet restore` and `dotnet publish` and you will be able to run your app from 
+the published directory. 
+
+## Resolving the Standard library packages
+The StdLib package is on a MyGet feed. In order to restore it, a MyGet feed needs to be added 
+to the NuGet feeds, either locally per application or in a central location. 
+
+**Issues tracking this:** 
+
+* [#535](https://github.com/dotnet/cli/issues/535)
+
+**Affects:** `dotnet restore`
+
+**Workaround:** update to the latest bits and run `dotnet new` in an empty directory. This will 
+now drop a `nuget.config` file that you can use in other applications. 
+
+If you cannot update, you can use the following `nuget.config`:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+    <clear />
+    <add key="dotnet-core" value="https://www.myget.org/F/dotnet-core/api/v3/index.json" />
+    <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
+  </packageSources>
+</configuration>
+```
+
+## Uninstalling/reinstalling the PKG on OS X
+OS X doesn't really have an uninstall capacity for PKGs like Windows has for 
```

Comment:
@richlander didn't. :) Yes and no, some would, some wouldn't. I thought, given the speed with which we are running, that it would be a good thing to have it somewhere...but, good point, maybe not in here.